Cargo-minimal.lock

```diff
@@ -86,7 +86,9 @@ dependencies = [
 name = "bitcoin-internals"
```

Comment:
Just to clarify the commit message, the feature gate is not for the macro itself - that would be an overkill, it's for `serde_json` and `bincode` dependencies, so that downstream users transitively depending on `internals` don't get random encodings they don't care about.